docs/index.rst

```diff
@@ -116,7 +116,7 @@ by this variable.
 Dependencies
 ------------
 
-``yarl`` requires :mod:`multidict` library.
+``yarl`` requires :mod:`multidict` library and ``propcache`` library.
```

Comment:
TODO: switch to `:mod:` once propcache is org transfered and published

Reply:
@bdraco I've configured RTD to use https://propcache.aio-libs.org. You can add it to the intersphinx mapping already.

Reply:
Thanks

Reply:
Looks like we need some more docs to make that work https://github.com/aio-libs/propcache/pull/16

Reply:
All looks good now